spec/authorities/qa/authorities/find_works_spec.rb

```diff
@@ -31,18 +31,18 @@
 
     context "when work has child works" do
       before do
-        work4.member_ids += [work1.id]
-        persister.save(resource: work4)
+        work1.member_ids += [work2.id]
```

Comment:
I think we should revert this change. and make line 39 just look for work2 similar to https://github.com/samvera/hyrax/blob/master/spec/authorities/qa/authorities/find_works_spec.rb#L37